### ISSUE TITLE
web-animations uncaught TypeError (a fix is included)

### ISSUE THREAD

**Issue by ArslanBouzid** [PredixDev/px-app-nav#29]
# Submit an Issue

Hi,

Thank you guys for having developed this useful component/wrapper.

Unfortunately, I've found a bug when using your component in addition to other polymer elements (paper-tooltip for example).

Your component has a file "px-app-nav.html" that uses this script:
**<script src="../web-animations-js/web-animations-next.min.js"></script>**

Whilst, paper-tooltip uses neon-animation that uses behind this script:
**<script src="../web-animations-js/web-animations-next-lite.min.js"></script>**

The errors I got:

- px-app-nav is not collapsable anymore and this error is displayed in the console:
**web-animations-next-animation.js:41 Uncaught TypeError: Cannot read property 'cancel' of null(…)**
- Multiple entries are highlighted in your component
- Other components are not working correctly (paper-tooltip for example) 

I fixed this issue by replacing in your file "px-app-nav.html" that specific line by this:
**<script src="../web-animations-js/web-animations.min.js"></script>**

This little modification fixed all the issues I got and had no side effects on your component.

Can you take a look at it ?

Cheers,

Arslan

## Steps to reproduce the problem:
Importing paper-tooltip or any other polymer component that uses web-animations-next-lite.

## Environment (_component version number, Browser (including version), operating system, hardware, etc_):
Chrome/Opera latest stable version
Latest (at this day) version of px-app-nav and paper-tooltip

**Comment by gedesigntech** [comment]
Hi @ArslanBouzid,

Unfortunately using web-animations-next.min.js uses SequenceEffect, which is undefined in web-animations.min.js.  This causes a ReferenceError in px-app-nav.

I wonder if there's a way to localize the web-animations-next-lite to paper-tooltip?

Related to this, we have in the past looked into using neon animations for px-app-nav, but were unable to see a neon animation that matched interactive behavior currently achieved through web-animations.  We are still looking to switch to neon when our requirements permit.

**Comment by mdwragg** [comment]
I wonder if we can move to `web-animations-next.min.js` for app nav? Might help smooth over the waters between Paper and our elements???

**Comment by ArslanBouzid** [comment]
Hello guys,

Thanks for the quick feedback.

It's indeed strange that when I replaced **web-animations-next.min.js** by **web-animations.min.js** in px-app-nav.html, everything went seamlessly:
- collapse feature was working
- no more uncaught exception
- all the other elements that used neon-animation ran correctly

Am I missing something ??

**Comment by eriepasquare** [comment]
+1  I've also run into the exact same error/conflict when mixing px and paper elements.  The fix proposed by ArslanBouzid also resolved the issue for me.

**Comment by ArslanBouzid** [comment]
Hello Guys,

Is there anything new related the fix I proposed and your plans ?

Thanks

**Comment by PawelRebisz** [comment]
Hi, I have faced exactly the same behavior. What are the plans to mitigate that and first of all, why this does work for `demo` and in application it does not.
`px-app-nav` component pulls directly the dependency, so I cannot understand how this is conflicting here.

**Comment by sjw787** [comment]
Hey I was just seeing the same behavior too. I made the replacement that @ArslanBouzid suggested and it has no problems. Is there any plan to incorporate that change or a similar change into the app-nav that addresses this issue?

**Comment by mdwragg** [comment]
Can the people that have got it working submit a pull request please? Thanks!

**Comment by manderson23** [comment]
px-vis 1.0.0 now imports neon-animation causing the following error when both px-app-nav and px-vis are used within the same application
```
TypeError: this._animation is null[Learn More]  web-animations-next.min.js:16:15334
  b.Animation http://localhost:5000/bower_components/web-animations-js/web-animations-next.min.js:16:15334
  _play http://localhost:5000/bower_components/web-animations-js/web-animations-next.min.js:16:14392
  Element.prototype.animate http://localhost:5000/bower_components/web-animations-js/web-animations-next.min.js:16:24226
  b.newUnderlyingAnimationForKeyframeEffect http://localhost:5000/bower_components/web-animations-js/web-animations-next-lite.min.js:16:12317
  _rebuildUnderlyingAnimation http://localhost:5000/bower_components/web-animations-js/web-animations-next-lite.min.js:16:4042
  b.Animation http://localhost:5000/bower_components/web-animations-js/web-animations-next-lite.min.js:16:3086
  _play http://localhost:5000/bower_components/web-animations-js/web-animations-next-lite.min.js:16:2179
  b.newUnderlyingAnimationForGroup http://localhost:5000/bower_components/web-animations-js/web-animations-next-lite.min.js:16:16929
  _rebuildUnderlyingAnimation http://localhost:5000/bower_components/web-animations-js/web-animations-next-lite.min.js:16:4245
  b.Animation http://localhost:5000/bower_components/web-animations-js/web-animations-next-lite.min.js:16:3086
  _play http://localhost:5000/bower_components/web-animations-js/web-animations-next-lite.min.js:16:2179
  play http://localhost:5000/bower_components/web-animations-js/web-animations-next-lite.min.js:16:2369
  _newAnimateMenuItem http://localhost:5000/bower_components/px-app-nav/px-app-nav.html.js:267:38
  markSelected http://localhost:5000/bower_components/px-app-nav/px-app-nav.html.js:338:13
  navClickHandler http://localhost:5000/bower_components/px-app-nav/px-app-nav.html.js:226:7
  handler http://localhost:5000/bower_components/polymer/polymer.html.js:550:1
	decorated
```
The error only occurs when you change selection moving up the list. There is no error when moving down the list.

**Comment by mdwragg** [comment]
Fixed by 435879a

**Comment by ArslanBouzid** [comment]
Thank you guys for the fix :) 
I can finally use your repo instead of my fork.
